src/styles/main.scss

```diff
@@ -14,16 +14,16 @@ body {
   display: grid;
   gap: 48px;
   @media (min-width: 1024px) {
-    grid-template-columns: repeat(4, 1fr);
+    grid-template-columns: repeat(4, 200px);
   }
   @media (min-width: 768px) and (max-width: 1023px) {
-    grid-template-columns: repeat(3, 1fr);
+    grid-template-columns: repeat(3, 200px);
   }
   @media (min-width: 488px) and (max-width: 767px) {
-    grid-template-columns: repeat(2, 1fr);
+    grid-template-columns: repeat(2, 200px);
   }
   @media (max-width: 488px) {
-    grid-template-columns: repeat(1, 1fr);
+    grid-template-columns: repeat(1, 200px);
   }
```

Comment:
Ok good, almost done. But width value is repeated. You can use variable for this.